frontend/src/metabase/visualizations/visualizations/Heading/index.ts

```diff
@@ -4,12 +4,13 @@ import {
   getDefaultSize,
   getMinSize,
 } from "metabase/visualizations/shared/utils/sizes";
+import type { VisualizationProperties } from "metabase/visualizations/types";
 import { Heading } from "./Heading";
 
 const HeadingWrapper = Object.assign(Heading, {
   uiName: t`Heading`,
   identifier: "heading",
-  iconName: "heading",
+  iconName: "heading" as any, // icon "heading" doesn’t exist
```

Comment:
TS actually helped here, if there is no such icon, we should not have `iconName` at all